app-info/metainfo/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec.metainfo.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="web-application">
+  <launchable type="url">https://community.endlessos.com/</launchable>
+  <url type="homepage">https://community.endlessos.com/</url>
+  <project_license>GPL-2.0-or-later</project_license>
+  <metadata_license>FSFAP</metadata_license>
+  <name>Endless</name>
```

Comment:
This should be "Endless Community". The site is not just called "Endless"

Reply:
Right. This is because the generator prefers `short_name` from the manifest if it exists. Our Discourse sets `name` to `Endless Community` and `short_name` to `Endless`. I briefly looked in the Discourse settings to see if that was specified but couldn't find anything. I'm going to add a `name_property` option that says whether you should use `name` or `short_name` since what's there is just a guess.

Reply:
I added support for this by letting the configuration specify `name_property`. However, maybe that's all moot if we switch to the approach where the generator just dumps out the boilerplate.